### PR TITLE
fix: allow annotations for service

### PR DESCRIPTION
**Pull request by ion-elgreco**
When you use an azure loadbalancer you need to be able to add an annotation on the service.

### CODE REVIEW

File: charts/lakefs/templates/service.yaml [hunk 1]
```diff
@@ -4,6 +4,8 @@ metadata:
   name: {{ include "lakefs.fullname" . }}
   labels:
     {{- include "lakefs.labels" . | nindent 4 }}
+  annotations:
```
Comment (Isan-Rivkin) [inline charts/lakefs/templates/service.yaml:7]: Hi @ion-elgreco first, THANK YOU for contributing! 

The change make sense, couple of change request: 
1. Notice the CI is failing, please bump the Chart `version` in `Chart.yaml`. **If** after that you see more errors in the CI make sure to fix or LMK if it's not clear why. (every merge release a new chart version automatically). 
2. in `values.yaml` please add a default under the `service` object, i.e 

```yaml
service:
  annotations: {}
  # existing values
  type: ClusterIP
  port: 80
```

**Review by Isan-Rivkin — CHANGES_REQUESTED** [review]

File: charts/lakefs/templates/service.yaml (hunk 1 above)
Comment (ion-elgreco) [reply charts/lakefs/templates/service.yaml:7]: @Isan-Rivkin done!

**Comment by Isan-Rivkin** [comment]
@ion-elgreco please bump version

**Comment by ion-elgreco** [comment]
@Isan-Rivkin done!

**Comment by Isan-Rivkin** [comment]
LGTM @ion-elgreco !

**Review by Isan-Rivkin — APPROVED** [review]

**Pull request merged**